core-services/release-controller/README.md

```diff
@@ -53,10 +53,14 @@ To abort/remove a release tag:
 
 * `deploy-*.yaml` - the controller definitions
 * `images-*.yaml` - definitions of the image streams used by the publish / build rules
-* `releases/release-*.json` - definitions of the build and publishing rules for sets of images
-* `repos/ocp-*-*.repo` - definitions where RPMs are loaded from for OCP builds
+* `_releases/release-*.json` - definitions of the build and publishing rules for sets of images
+* `_repos/ocp-*-*.repo` - definitions where RPMs are loaded from for OCP builds
 * `rpms-*.yaml` - definitions of the services that exposed mirrored RPMs within the clutser
 
+### Release informers
+
+Release informers are configured via the `verify` property in `_releases/release-*.json`. To update TestGrid after making changes, use [`testgrid-config-generator`][testgrid-config-generator].
```

Comment:
i think the config really starts here:
https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml#L297

not sure how it finds its way into release-controller though

Reply:
openshift/ci-tools#632 is an attempt to clarify how the pieces get collected.  With that in place, I think this PR is good to go as it stands.